ingest.go

```diff
@@ -1359,6 +1365,7 @@ func (d *DB) ingest(
 	targetLevelFunc ingestTargetLevelFunc,
 	shared []SharedSSTMeta,
 	exciseSpan KeyRange,
+	doFlushableIngest bool,
```

Comment:
I think this should be called `localFilesContainExciseSpanTombstones`  or `sstsContainExciseTombstone` or something with an explanation of the requirement on the local sstable(s) containing a rangedel/rangekeydel spanning the entirety of the excise span. `doFlushableIngest` isn't appropriate as it also can't _force_ a flushable ingest to happen either.